lua/astrocommunity/editing-support/auto-save-nvim/auto-save-nvim.lua

```diff
@@ -0,0 +1,5 @@
+return {
+  "Pocco81/auto-save.nvim",
+  lazy = false,
+  config = function() require("auto-save").setup {} end,
```

Comment:
any reason why you don't use `opts = {}`

Reply:
Moved to suggestion type comment.